troupon/cart/views.py

```diff
@@ -43,3 +50,78 @@ def post(self, request, **kwargs):
         }
         return render(request, self.template_name, context)
 
+
+class AddToCartView(LoginRequiredMixin, View):
+    """Adds item to cart.
+
+    Returns:
+        A redirect to the deals homepage
+    """
+
+    def get(self, request, **kwargs):
```

Comment:
**Code Quality: Nitpick**

You should move this to another view named `CartView` or `ViewCart` which will be for the sole purpose of viewing the cart.